index.html

```diff
@@ -358,7 +373,7 @@ <h2>
                 <input type="text" name="name" class="hover" maxlenght="30" placeholder="Full name" required>
                 <input type="email" name="email" class="hover" placeholder="Email address" required>
                 <textarea type="text" name="text" class="hover" maxlenght="500" placeholder="Enter text here" required></textarea>
-                <button class="btngreen">Get in touch</button>
+                <button type="button" class="btngreen">Get in touch</button>
             </form>
         </div>
         <div class="img2"></div>
```

Comment:
- [ ] For the recent `Form` section, kindly make this section look identical to that of the Figma template you chose. To highlight the changes to be made in this section, you need to:

- The `form` and the `p` elements should be on a row. The `form` element should be on the right side while the `p` element with the text `"I'm always interested in hearing ..."` should be on the left side. 💻 
- The background images used should be present and well aligned
- The social media icons are at the center and aligned with the correct margin between each icon 👍 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/208155177-c12f4702-923c-48c9-a5cf-7fee2312d5a8.png)

### Figma Design

![image](https://user-images.githubusercontent.com/77106962/208155268-1decc056-d6f1-43dd-9aa7-e2ae734d66ba.png)

#### P.S: The form input for the full name is two (first and last name) for the desktop version of the design on Figma. However, due to the requirement, you should have one input for the full name, so you do not have to implement it the same way it is implemented on Figma.

![image](https://user-images.githubusercontent.com/77106962/208155847-a2f95930-f82c-4625-a5de-95354b69c419.png)